src/test/vat.t.sol

```diff
@@ -522,13 +525,29 @@ contract BiteTest is DSTest {
         me = address(this);
     }
 
+    function test_set_lump_multiple_ilks() public {
+        cat.file("gold",   "lump", rad(115792 ether));
+        (,, uint256 goldLump) = cat.ilks("gold");
+        assertEq(goldLump, rad(115792 ether));
+        cat.file("silver", "lump", rad(115792 ether));
+        (,, uint256 silverLump) = cat.ilks("gold");
```

Comment:
This should be `(,, uint256 silverLump) = cat.ilks("silver");`

Reply:
We should also make different `lump`s for the different ilks so we can assert that they're different

Reply:
good catch